src/routes/MoreInformation.tsx

```diff
@@ -1,88 +1,106 @@
 import React from 'react';
+import { Translate } from '../components/Localization';
 import { TextHeadingPage } from '../components/HeadingPageLayouts';
 import NoTestFound from '../components/Result/NoTestFound';
-import { useTestData } from '../utils/hooks';
 import TestFoundButNoData from '../components/Result/TestFoundButNoData';
 import { BigLinkButton } from '../components/Buttons';
-function getReadableAntigen(antigen: string): string {
-  if (antigen == 'N') {
-    return 'Nucleoprotein';
-  } else if (antigen == 'S') {
-    return 'Spike';
-  } else if (antigen == 'N+S') {
-    return 'Nucleoprotein + Spike';
-  }
-  return antigen;
-}
+import { useTestData } from '../utils/hooks';
+
+const antigenMap: Record<string, string> = {
+  N: 'Nucleoprotein',
+  S: 'Spike',
+  'N+S': 'Nucleoprotein + Spike',
+};
+
+const getReadableAntigen = (antigen: string) => antigenMap[antigen] || antigen;
```

Comment:
I think you missed a translate call here?

Reply:
would consider it a technical term and not translate it